### PR TITLE
Python testing scripts.

### PR DESCRIPTION
**Pull request by miskosz**
Add python testing.

I'm not that confident with bash, comments are welcome.

Also I added the `__init__.py` files, I think it is the simplest and most reliable solution.

Note: Still setting up Travis.

### CODE REVIEW

**Comment by lukipuki** [comment]
I think for Travis we can skip virtualenv and just use pip. The virtual machine gets thrown away anyway. What would be the added value of using virtualenv there? (The only one I know of setting up $PATH which we don't use)

**Comment by miskosz** [comment]
It is useful to run tests in local development.
(Also in general it is nicer, say e.g. if conan had incompatible setup.)

**Comment by lukipuki** [comment]
I'm not disputing local development, just that we can skip setting up virtualenv in Travis.

**Comment by miskosz** [comment]
Ready for review.

I thought I'd reuse the venv in CI, in the end it is useful just for developing locally. Also we duplicate the package installation now in 3 places:
* dockerfile
* travis
* create venv

I think it might be difficult to consolidate, so I'd just keep it as is for now.

File: covid_graphs/README.md [hunk 1]
```diff
@@ -11,21 +11,31 @@ You can then access the server locally at [localhost:8081](http://127.0.0.1:8081
...
-## Running standalone graphs
+## Local development
 
-For quick development or data examination, running standalone graphs is useful.
+This script creates a virtual environment with all development dependencies.
+Don't forget to activate the environment:
+```sh
+./create_dev_venv.sh
+source .venv/bin/activate
+```
```
Comment (lukipuki) [inline covid_graphs/README.md]: I think this order is more clear.
```suggestion
```sh
./create_dev_venv.sh
source .venv/bin/activate # Don't forget to activate the environment
```
```

File: covid_graphs/create_dev_venv.sh [hunk 2]
```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+set -euo pipefail
```
Comment (lukipuki) [inline covid_graphs/create_dev_venv.sh:2]: I think this is the same as
```suggestion
#!/usr/bin/env bash -eu
set -o pipefail
```

I've been using `bash -e` and I'm not familiar with `-u`. Perhaps even `-o pipefail` can be put there, e.g. `!/usr/bin/env bash -euo pipefail`

**Review by lukipuki — APPROVED** [review]

File: covid_graphs/create_dev_venv.sh (hunk 2 above)
Comment (miskosz) [reply covid_graphs/create_dev_venv.sh:2]: I took inspiration from our codebase :) This is what "the internets" say: https://vaneyckt.io/posts/safer_bash_scripts_with_set_euxo_pipefail/

Comment (miskosz) [reply covid_graphs/create_dev_venv.sh:2]: I pipefail can't go there, so I've kept it as it was.

**Pull request merged**